frontend/src/components/mine/Compliance/MineComplianceInfo.js

```diff
@@ -24,90 +25,187 @@ const defaultProps = {
-                        {moment(this.props.mineComplianceInfo.last_inspection).format(
-                          "MMM DD YYYY"
-                        )}
-                      </p>
-                      <p className="info-display-label">LAST INSPECTION DATE</p>
-                    </div>
-                  </Col>
-                  <Col span={4} />
-                  <Col span={4}>
-                    <div className="center">
-                      <p className="info-display">
-                        {this.props.mineComplianceInfo.num_overdue_orders}
-                      </p>
-                      <p className="info-display-label">OVERDUE ORDERS</p>
-                    </div>
-                  </Col>
-                  <Col span={4} />
-                  <Col span={4}>
-                    <div className="center">
-                      <p className="info-display">
-                        {this.props.mineComplianceInfo.num_open_orders}
-                      </p>
-                      <p className="info-display-label">OPEN ORDERS</p>
-                    </div>
-                  </Col>
-                  <Col span={2} />
-                </Row>
-                <br />
-                <br />
-                <Row gutter={16} justify="center" align="top">
-                  <Col span={2} />
-                  <Col span={4}>
-                    <div className="center">
-                      <p className="info-display">{this.props.mineComplianceInfo.warnings}</p>
-                      <p className="info-display-label">WARNINGS ISSUED IN THE PAST YEAR</p>
-                    </div>
-                  </Col>
-                  <Col span={4} />
-                  <Col span={4}>
-                    <div className="center">
-                      <p className="info-display">{this.props.mineComplianceInfo.advisories}</p>
-                      <p className="info-display-label">ADVISORIES ISSUED IN THE PAST YEAR</p>
-                    </div>
-                  </Col>
-                  <Col span={4} />
-                  <Col span={4}>
-                    <div className="center">
-                      <p className="info-display">
-                        {this.props.mineComplianceInfo.section_35_orders} - Section 35
-                      </p>
-                      <p className="info-display-label">TOTAL</p>
+        <div>
+          {this.state.isLoading && <Loading />}
+          {!this.state.isLoading && (
+            <div>
+              <h4>Compliance Overview</h4>
+              <br />
+              <br />
+              {this.props.mineComplianceInfo && (
+                <div>
+                  <Row gutter={16} justify="center" align="top">
+                    <Col span={2} />
+                    <Col span={4}>
+                      <div className="center">
+                        <p className="info-display">
+                          {moment(this.props.mineComplianceInfo.last_inspection).format(
```

Comment:
I created a utils function to handle this formatting. Probably best to update components to use that as we touch them to centralize our formatting and use of momentjs.

Reply:
I talked to Tatianna and swapped all the dates over to use this function.